scripts/protocgen.sh

```diff
@@ -18,28 +18,24 @@ protoc_gen_doc() {
 protoc_gen_gocosmos
 protoc_gen_doc
 
-proto_dirs=$(find ./proto -path -prune -o -name '*.proto' -print0 | xargs -0 -n1 dirname | sort | uniq)
-# TODO: migrate to `buf build`
-for dir in $proto_dirs; do
-  buf alpha protoc \
-  -I "proto" \
-  -I "third_party/proto" \
-  --gocosmos_out=plugins=interfacetype+grpc,\
-Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types:. \
-  --grpc-gateway_out=logtostderr=true:. \
-  $(find "${dir}" -maxdepth 1 -name '*.proto')
-
-done
-
-# command to generate docs using protoc-gen-doc
-# TODO: migrate to `buf build`
-buf alpha protoc \
--I "proto" \
--I "third_party/proto" \
---doc_out=./docs/protocol \
---doc_opt=./docs/protodoc-markdown.tmpl,proto-docs.md \
-$(find "$(pwd)/proto" -maxdepth 5 -name '*.proto')
+# prepare for swagger
+mkdir -p ./tmp-swagger-gen
+
+# protoc and swagger
+buf build
```

Comment:
this should only build the `proto/` directory, not `third_party/`

Reply:
fixed so that only the `proto/` directory, and files necessary from the `third_party/` directory (for `swagger-combine`) are built.